spec/eof.md

```diff
@@ -348,6 +396,43 @@ During scanning, for each instruction:
 
 Annotated examples of EOF formatted containers demonstrating several key features of EOF can be found in [this test file within the `evmone` project repository](https://github.com/ethereum/evmone/blob/master/test/unittests/eof_example_test.cpp).
 
+## Appendix: Creator Contract
+
+```solidity
+{
+    /// Takes [tx_initcode_hash][salt][init_data] as input,
+    /// creates contract and returns the address or failure otherwise
+
+    // init_data.length can be 0, but the first 2 words are mandatory
+    let size := calldatasize()
+    if lt(size, 64) { revert(0, 0) }
+
+    let tx_initcode_hash := calldataload(0)
+    let salt := calldataload(32)
```

Comment:
`salt` variable is not used anymore

Reply:
Also perhaps better to move `tx_initcode_hash` to after hashing, closer to where it's used in txcreate()

Reply:
done both here and in the eip, thx